.storybook/helpers/h5p.utils.ts

```diff
@@ -45,43 +45,189 @@ export const semantics: H5PField = {
         fields: [
           { name: "id", type: H5PFieldType.Text, widget: "hidden" },
           {
-            name: "xPercentagePosition",
-            type: H5PFieldType.Number,
-            widget: "hidden",
+            label: "Start Date",
+            name: "start_date",
```

Comment:
These are camelCased in `semantics.json`
```suggestion
            name: "startDate",
```